README.md

```diff
@@ -16,10 +16,10 @@ and more...
 
 As a learner in social sciences, it is really the time to study computational methods.
 
-##Things I have learned so far
+## Things I have learned so far
 How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
```

Comment:
I really like the use of `` here to refer to fork as a function!

Reply:
I second the use of ``, I always wondered how people made the text look that way and now I know!